src/app/ui/bridge/utils.tsx

```diff
@@ -17,7 +17,13 @@ export function buildExplorerLink(
       color="text.primary"
       component={Link}
       target="_blank"
-      href={`${blockExplorerUrls[0]}/tokens/${address}`}
+      href={`${blockExplorerUrls[0]}tokens/${address}`} // todo: on OP it needs to be "token/..."
```

Comment:
do we need the ocmment ?

Reply:
Yes, I removed the comment for now. But note: I double checked for multiple blockchain explorers on how the token-links work and removed and it turns out:
- all chains I have checked, the url for token infos is "token" as singular instead of "tokens"
- we also did have another issue as the blockExplorerUrls[0] already comes with a trailing-slash. Because we added another one, we ended up with "//" in the link

As a note: I did not check across all blockchains if the "/token" works as a general solution